libefp/default.nix

```diff
@@ -0,0 +1,76 @@
+{ stdenv, fetchFromGitHub, cmake, gfortran
+# Dependencies
+, blas
+, lapack
+# Config
+, fraglib_deep ? false
+, fraglib_underscore_l ? false
+, psi4_patches ? true
+}:
+assert
+  stdenv.lib.asserts.assertMsg
+  (!blas.isILP64)
+  "32 bit integer BLAS implementation required.";
+
+stdenv.mkDerivation rec {
+    pname = "libefp";
+    version = if psi4_patches
+      then "1.5.0-psi4"
+      else "1.5.0";
+
+    nativeBuildInputs = [
+      cmake
+      gfortran
+    ];
+
+    buildInputs = [
+      blas
+      lapack
+    ];
+
+    src =
+      let
+          rev = if psi4_patches
+                  then "15cd7ce91239c04b5c32ed101bde6cc36c57550a"
+                  else version;
+          sha256 = if psi4_patches
+                     then "1zlihc7hp2vcdcwi82n7xg41bm1vv7n1mx1jdff4w1amhg9rlahm"
+                     else "0jcvl3chni4f0hddx9blaia3kccfqx7cszrwavp0a35d42n0x5i2";
```

Comment:
The hashes are maybe swaped? I get:
```
unpacking source archive /build/1.5.0.tar.gz
hash mismatch in fixed-output derivation '/nix/store/ykj5x09a65f6ymyiapmv1zxv0jdlbvmg-source':
  wanted: sha256:0jcvl3chni4f0hddx9blaia3kccfqx7cszrwavp0a35d42n0x5i2
  got:    sha256:1zlihc7hp2vcdcwi82n7xg41bm1vv7n1mx1jdff4w1amhg9rlahm
```

Reply:
You are right. They were. I must have messed up completely with all the hashes when trying to merge everything into one file. Confirmed with `nix-prefetch-url` and should be fixed now by [fc855a8](https://github.com/markuskowa/NixOS-QChem/pull/6/commits/fc855a8168bcab13bcb2ab74deb65b844047c6c8).